src/app/components/Snapshots/SnapshotCardDurationLabel.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react'
+import { TFunction } from 'i18next'
+import { useTranslation } from 'react-i18next'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import { ChartDuration } from '../../utils/chart-utils'
+
+const getLabels = (t: TFunction): Record<ChartDuration, string> => ({
+  [ChartDuration.TODAY]: t('chartDuration.lastHour'),
+  [ChartDuration.WEEK]: t('chartDuration.lastDay'),
+  [ChartDuration.MONTH]: t('chartDuration.lastDay'),
+  [ChartDuration.ALL_TIME]: t('chartDuration.lastDay'),
```

Comment:
Hm, looks like Active Accounts shows value for last month:
![image](https://github.com/oasisprotocol/explorer/assets/3758846/55c30e99-aced-490b-bb6f-5c7dc6522191)

Reply:
Oh yeah, will ask Don what he wants here. It's weird that we have different behaviours for "all time" filter.

Reply:
Ok, charts behave correctly according to Don. 
"Exactly, they don’t have to be the same because we are showing different increments in the two charts"
We want two diff labels.

Reply:
I would rather switch to showing both daily, since we also can't correctly aggreggate monthly active accounts :shrug: